### ISSUE TITLE
Error command in installation guild

### ISSUE THREAD

**Issue by GingoBang** [tensorflow/tensorflow#17614]
Please go to Stack Overflow for help and support:

https://stackoverflow.com/questions/tagged/tensorflow

There is an small error in the [installation guild](https://www.tensorflow.org/install/install_mac#determine_which_tensorflow_to_install)
which is 7th step under the **Installing with Virtualenv** section.

The site give an example command of installing TensorFlow in the active Virtualenv for macOS, python which is actually for py3 with pip3 command. 
```
 $ pip3 install --upgrade \
 https://storage.googleapis.com/tensorflow/mac/cpu/tensorflow-1.6.0-py3-none-any.whl
```

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by yongtang** [comment]
Added a PR #17614 for the doc fix.

**Comment by asimshankar** [comment]
I believe this was fixed by https://github.com/tensorflow/tensorflow/commit/f4e70be18b104fbb2efeefeb83bea190aec12727
but an update to the site hasn't been pushed since.

@MarkDaoust : Can we push an update?

**Comment by yongtang** [comment]
@asimshankar @MarkDaoust It looks like the issue was fixed by f4e70be, but later it has been overridden by 9dae88d. I created #17617 to get the fix back.
